apps/meteor/app/api/server/v1/users.ts

```diff
@@ -1240,6 +1241,27 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.isMuted',
```

Comment:
I'm wondering if we should really implement this as a `users` endpoint or into somewhere else :thinking: What made me think about this is that we do not send a `roomId` in any other user related endpoint
This does looks ok to me as it is, but maybe having something such as `rooms.isMuted` or `subscriptions.isMuted` would look better since this is not related to the user in a global context. Would like to hear others' opinions, though
cc @MarcosSpessatto @gabriellsh

Reply:
Yes, makes sense, I was also skeptical about this. 

Moving this endpoint to Subscription makes more sense, I'll do that. Thanks!

Reply:
Done!